frog/imports/client/Wiki/index.js

```diff
@@ -185,7 +189,8 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     wikiStore.setPages(this.wikiDoc.data.pages);
     this.setState({
       pagesData: wikiStore.pages,
-      settings: this.wikiDoc.data.settings
+      settings: this.wikiDoc.data.settings,
+      isOwner: this.wikiDoc.data.owners.find(x => x === Meteor.userId()) === undefined
```

Comment:
Should it be `!== undefined`, meaning we found a match between the current user and one of the owners of the wiki?